package com.Exception;

public final class Finally_block {

	public static void main(String[] args) {
			int a = 10;
			System.out.println("Variable");
			System.err.println(a);
			
			System.out.println(a);
			
			System.exit(0);		//terminate 
			System.out.println("after printing a");
	}

}
